src/main/minio.js

```diff
@@ -1194,29 +1194,47 @@ export class Client {
     if (!isString(marker)) {
       throw new TypeError('marker should be of type "string"')
     }
-    if (!isString(delimiter)) {
+    let{
+      Delimiter,
+      MaxKeys,
+      IncludeVersion,
+    } = listQueryOpts
+
+    if (!isObject(listQueryOpts)) {
+      throw new TypeError('listQueryOpts should be of type "object"')
+    }
+   
+    if (!isString(Delimiter)) {
       throw new TypeError('delimiter should be of type "string"')
     }
-    if (!isNumber(maxKeys)) {
+    if (!isNumber(MaxKeys)) {
```

Comment:
The name of the variable in the `throw` message doesn't match after the renaming.